<?xml version="1.0"?>
<launch>
  <rosparam param="/vive/world_offset">[0, 0, 2]</rosparam>
  <rosparam param="/vive/world_yaw">0.0</rosparam>
  <node name="vive_node" pkg="vive_ros" type="vive_node" output="screen" required="true"/>
</launch>
